layouts/default.vue

```diff
@@ -38,36 +38,48 @@
       </v-list>
     </v-navigation-drawer>
     <v-app-bar fixed app>
```

Comment:
This got me thinking to use `<v-app-bar color="primary">`, and then perhaps the menu buttons with `color="secondary"`. 

What do you think? Worth playing around and seeing how it looks? Otherwise, no change necessary in the PR.

Reply:
I like how the cobalt blue app bar and sky blue buttons look. However the dark text of the WOUDC title is difficult to see when using a dark app bar. Could we change the color of this text to white or something lighter?

Reply:
Yep, that's possible.
For the text portion, you can specicy the class `{colourName}--text`. Colour names from here: https://vuetifyjs.com/en/styles/colors/#material-colors

And optionally if you want to specify a particular shade of the colour, use : `text--darken-{n}` or `text--lighten-{n}`, where `n` is a value between 1-4.